fala/templates/adviser/results.html

```diff
@@ -57,14 +57,17 @@ <h2 class="govuk-body-l govuk-!-font-weight-bold">{{ item.organisation.name }}</
             {% endif %}
             {% if item.categories|length %}
             <div class="categories">
-              <h3 class="govuk-heading-s govuk-!-margin-bottom-2">{{ _('Categories') }}</h3>
+              <h3 class="govuk-heading-s govuk-!-margin-bottom-2">{{ _('Categories ') }}</h3>
               <ul class="govuk-list govuk-list--bullet">
                 {% for cat in item.categories %}
-                  <li>{{ cat }}</li>
+                  <li >{{ cat }}</li>
```

Comment:
```suggestion
                  <li>{{ cat }}</li>
```

Reply:
thanks - I have no idea how that artifact got in ther